jet_tracking/jet_control.py

```diff
@@ -123,15 +126,17 @@ def calibrate(injector, camera, params, offaxis=False):
     # collect images and motor positions to calculate pxsize and cam_roll
     imgs = []
     positions = []
-    start_pos = injector_axis.get()
+    start_pos = injector_axis.user_readback.get()
     for i in range(2):
         image = get_burst_avg(20, camera.image)
         imgs.append(image)
-        positions.append(injector_axis.get())
-        injector_axis.put(injector_axis.get() - 0.1)
-        sleep(3)
-    injector_axis.put(start_pos)
-    sleep(3)
+        positions.append(injector_axis.user_readback.get())
+        next_position = injector_axis.user_setpoint.get() - 0.1
+        injector_axis.set(next_position, wait=True)
+        sleep(delay)
+
+    injector_axis.set(start_pos, wait=True)
```

Comment:
There was something where if the moves were too small on the injector  and we actually didn't see the `DMOV` flag flip. I don't know if this was here, or if this is what the magic sleep is here for. This just jogged my memory.

Reply:
Does the SLAC modified-motor-record/IMS implementation not guarantee a toggling of DMOV... ? Subscriptions callbacks shouldn't be getting lost here.

Reply:
No, but if the move is done before you subscribe. Digging through emails... It might have been a speed of request issue more than a distance issue. 

@ZLLentz do you remember any of this?

Reply:
I thought this worked fine, I'll test again

Reply:
I do know that the device_states records skip all processing if you move to the current state, but I don't remember this applying to small moves in the motor itself

Reply:
FWIW we should fix the IMS record if we find stuff like this instead of always chasing our tails

Reply:
Definitely. Would be happy to help figure it out, if needed.